htdocs/core/class/notify.class.php

```diff
@@ -1218,10 +1218,14 @@ public function send($notifcode, $object, $filename_list = array(), $mimetype_li
 					$emailTemplate = $formmail->getEMailTemplate($this->db, $object_type.'_send', $user, $outputlangs, 0, 1, $mailTemplateLabel);
 				}
 				if (!empty($mailTemplateLabel) && is_object($emailTemplate) && $emailTemplate->id > 0) {
-					if (property_exists($object, 'thirdparty') && $object->thirdparty instanceof Societe && $object->thirdparty->default_lang && $object->thirdparty->default_lang != $langs->defaultlang) {
-						$outputlangs = new Translate('', $conf);
-						$outputlangs->setDefaultLang($object->thirdparty->default_lang);
-						$outputlangs->loadLangs(array('main', 'other'));
+					if (property_exists($object, 'thirdparty')) {
+						$object->fetch_thirdparty();
```

Comment:
Weshould avoid this if $object->thirdparty is already loaded (already instanceof Societe).